### PR TITLE
Add Art & Architecture category to CreateWiki

### PR DESCRIPTION
**Pull request by dmehus**
We do get quite a few wikis that propose to create either a wiki about buildings and other real world infrastructure. Combining with "Art", as is common, also means this could potentially be used for the many artist repository wikis or even fan-made original character repositories/catalogues, too. It's also a common categorization

### CODE REVIEW

File: LocalSettings.php
Comment (RhinosF1) [inline LocalSettings.php:704]: - should be an =

**Review by RhinosF1 — CHANGES_REQUESTED** [review]

File: LocalSettings.php
Comment (dmehus) [reply LocalSettings.php]: @RhinosF1 Yeah, I noticed that _just_ before you replied, so already fixed it.

**Review by RhinosF1 — APPROVED** [review]

File: LocalSettings.php
Comment (RhinosF1) [reply LocalSettings.php]: Seen the fix and approved the change after that change.

Comment (dmehus) [reply LocalSettings.php]: Wait, don't merge this yet. I noticed another typo.

Comment (dmehus) [reply LocalSettings.php]: Fixed.

**Pull request merged**
